internal/controllers/monitoring/monitoring_test.go

```diff
@@ -0,0 +1,206 @@
+package monitoring_test
+
+import (
+	"context"
+	"fmt"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	api "github.com/ydb-platform/ydb-kubernetes-operator/api/v1alpha1"
+	testobjects "github.com/ydb-platform/ydb-kubernetes-operator/e2e/tests/test-objects"
+	"github.com/ydb-platform/ydb-kubernetes-operator/internal/controllers/database"
+	"github.com/ydb-platform/ydb-kubernetes-operator/internal/controllers/monitoring"
+	"github.com/ydb-platform/ydb-kubernetes-operator/internal/controllers/storage"
+	"github.com/ydb-platform/ydb-kubernetes-operator/internal/labels"
+	"github.com/ydb-platform/ydb-kubernetes-operator/internal/test"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"path/filepath"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"testing"
+)
+
+var (
+	k8sClient client.Client
+	ctx       context.Context
+)
+
+func TestMonitoringApis(t *testing.T) {
+	RegisterFailHandler(Fail)
+
+	test.SetupK8STestManager(&ctx, &k8sClient, func(mgr *manager.Manager) []test.Reconciler {
+		return []test.Reconciler{
+			&monitoring.DatabaseMonitoringReconciler{
+				Client: k8sClient,
+				Scheme: (*mgr).GetScheme(),
+			},
+			&monitoring.StorageMonitoringReconciler{
+				Client: k8sClient,
+				Scheme: (*mgr).GetScheme(),
+			},
+		}
+	})
+	BeforeEach(func() {
+		nsName := testobjects.YdbNamespace
```

Comment:
Very minor, but I kind of feel that this code about namespace initialization is pretty much identical (in medium tests, as in here, as well as in large tests). Can we maybe isolate this piece of code somewhere to test utilities (we probably don't have test utilities yet)?